scripts/installer.sh

```diff
@@ -541,7 +541,7 @@ main() {
 			;;
 	esac
 
-	echo "Installation complete! Log in to start using Tailscale by running:"
+	echo "Installation complete! Login to start using Tailscale by running:"
```

Comment:
Changed

Reply:
You changed it the wrong way. It was correct before. The verb has a space in it. "login" is a noun.